### ISSUE TITLE
Connection to chat server from Mobile/Web

### ISSUE THREAD

**Issue by nmarazov** [MUnique/OpenMU#124]
I have an idea for mobile app or web to connect with acc credentials to chat server and communicate with your friends and guild. From my days playing MU it was so hard to communicate and organize events if you didn`t have everyone skype. This way the game will be more engaging and fun. The chat server will need to be wrapped and use OpenApi for easier implementation of different chat clients. I have experience with Xamarin Mobile development and a simple mobile app for both iOS and Android won`t be so difficult.
What do you think of this idea?

**Comment by MarioGK** [comment]
Maybe we could use some already made communication/messagin protocols like IRC or XMPP, instead of making everything from zero.

**Comment by sven-n** [comment]
Thanks, I had some ideas in this direction as well. I’d rather make it possible to use existing chat platforms/protocols like XMPP, IRC or to connect with discord. There are plenty of available clients for that.

**Comment by MarioGK** [comment]
I think connecting to a discord server will be the best choice, it is more popular than IRC and XMPP, and it is cross platform.

**Comment by sdyalor** [comment]
> 
> 
> Maybe we could use some already made communication/messaging protocols like IRC or XMPP, instead of making everything from zero.

MQTT

**Comment by Leonardo-Ferreira** [comment]
is this still relevant?

**Comment by sven-n** [comment]
> is this still relevant?

Yes, but I don't have a good idea how to do it yet 😅 If you have, feel free 😄

**Comment by nitoygo** [comment]
problem with this is each character has their own 'persona' (iykyk), but they share a common login. if in discord, each user would have to login a different account per character.
